webrtc.html

```diff
@@ -5519,13 +5518,10 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
               <p><code>getParameters</code> may be used with
               <code>setParameters</code> to change the parameters in the
               following way:</p>
-              <pre class="example highlight">
-              var params = sender.getParameters();
+              <pre class="example highlight">const params = sender.getParameters();
```

Comment:
Is there a reason for starting/ending the example code on the same line as the `<pre>`/`</pre>` tags? I think it's more readable the other way.

Reply:
`<pre>` shows input as-is, so if adding extra line break between tag and code, it shows extra space in rendered HTML.
like below (L: before, R: after)

<img width="1650" alt="screen shot 2017-10-13 at 16 04 58" src="https://user-images.githubusercontent.com/322289/31534091-58ba51ae-b030-11e7-8133-e91cec19db94.png">

Reply:
Hmm, I tested with Chrome and Firefox and don't see a difference.

The [HTML5 spec](https://www.w3.org/TR/html5/grouping-content.html#the-pre-element) says:

> a leading newline character immediately following the pre element start tag is stripped

> A single newline may be placed immediately after the start tag of pre and textarea elements. If the element's contents are intended to start with a newline, two consecutive newlines thus need to be included by the author.

So I wonder how you were able to get different behavior?

Reply:
I found that this caused by `<code>` after `<pre>` only inserted in Example 1 and 2.
other example doesn't has `<code>` so no effect for line break.
(this `<code>` seems for build highlighting and remove after render)

I'll insert break line except Ex 1, 2.